docs/UserGuide.adoc

```diff
@@ -72,6 +72,11 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sorting all persons by name : `sort`
+
+Shows a sorted list of all persons, along with their non-private details, in the address book. +
```

Comment:
While it may be a bit pedantic, it would be better to indicate which field is being sorted and what criteria it is being sorted by.